src/exercises/chromium/third-party.md

```diff
@@ -8,6 +8,8 @@ to handle untrustworthy input.
 skip ahead and do that now if you like. Or, you could create a new
 [`rust_executable` target][2] which uses `uwuify`).
 
+(Thanks to [@daniel_c0deb0t][3] for this crate!)
```

Comment:
Also, since the course is a programming resource, I would prefer linking to a GitHub profile. We've done that in the past for people who have contributed directly to the course: so far only on https://google.github.io/comprehensive-rust/running-the-course/translations.html.

Reply:
Done